x-pack/plugins/security_solution/common/endpoint/service/trusted_apps/validations.ts

```diff
@@ -34,6 +34,24 @@ export const getDuplicateFields = (entries: ConditionEntry[]) => {
     .map((entry) => entry[0]);
 };
 
+const WIN_EXEC_PATH = /\\(\w+|\w+[\w+|-]+\/ )+\w+[\w+|-]+\.*\w+$/i;
+const UNIX_EXEC_PATH = /(\/|\w+[\w+|-]+\\ )+\w+[\w+|-]+\.*\w*$/i;
```

Comment:
🤩 

Could you add a code comment describing what these RegExp are supposed to match on... because I'm dizzy looking at it 😆

Reply:
Fair point!😆

Reply:
done bb7f1d07f7c216ac7ecf3cc9bbbf20ebb51debf6